### PR TITLE
[naming] prefix packages with sqooba

### PR DESCRIPTION
**Pull request by zouzias**
To test:

```
cookiecutter https://github.com/Sqooba/cookiecutter-pypackage --checkout feature/prefix-packages-with-sqooba
```

Pay attention to the project name used, say `foo`.

To verify (in the root directory of the project):

```
pip install -r requirements_dev.txt (this will fixed in a diffent PR)
make test # should work
```

```
sqooba-foo
```
should output

```
Replace this message by putting your code into sqooba.foo.cli.main
See click documentation at http://click.pocoo.org/
```

### CODE REVIEW

**Comment by tukistukistukis** [comment]
We need to:
-Fix make servedocs
-Push the new changes in make coverage

File: {{cookiecutter.project_slug}}/setup.py [hunk 1]
```diff
@@ -35,19 +35,13 @@
         '{{ license_classifiers[cookiecutter.open_source_license] }}',
 {%- endif %}
         'Natural Language :: English',
-        "Programming Language :: Python :: 2",
-        'Programming Language :: Python :: 2.7',
-        'Programming Language :: Python :: 3',
-        'Programming Language :: Python :: 3.4',
-        'Programming Language :: Python :: 3.5',
-        'Programming Language :: Python :: 3.6',
         'Programming Language :: Python :: 3.7',
```
[the remark this answers is not in the document]
Comment (zouzias) [reply {{cookiecutter.project_slug}}/setup.py]: I put back Python 3.6, let's leave support for 3.6/3.7 for now

**Comment by zouzias** [comment]
> See comments above in the conversation

`make servedocs` and `make coverage` should work like a charm now. Can you re-test?

**Comment by tukistukistukis** [comment]
Yes it works! very cool :)

**Comment by zouzias** [comment]
> Yes it works! very cool :)

Thanks for the review!

**Pull request merged**